static/js/app.js

```diff
@@ -89,7 +89,7 @@ async function sendForm(formAction, formData) {
 }
 
 function copyToClipboardAndShowMessage(triggeringEl) {
-  const aliasCopyBtn = triggeringEl;
+  // const aliasCopyBtn = triggeringEl;
```

Comment:
Should we just remove this line?
```suggestion
```

Reply:
@groovecoder Dang! I should've commented here. Neither this (or the comment) below was actually in use. I think we can remove them. I just didn't wanna mess anything up.